### PR TITLE
Feature/121 Implementing menu feature

### PR DESCRIPTION
**Pull request by Mnx458**

### CODE REVIEW

File: bike-erp/src/app/App.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
  .App {
-  width:100%;
+  width: 100%;
   height: 100%;
   text-align: center;
-  padding: 20px;
-  box-sizing: border-box;
+  /* padding-top: 60px; */
```
Comment (nasaku898) [inline bike-erp/src/app/App.css:5]: remove comment

File: bike-erp/src/components/Menu/ERPMenu.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.navbar {
```
Comment (nasaku898) [inline bike-erp/src/components/Menu/ERPMenu.css:1]: I'd recommend switching to `Material UI` styling

File: bike-erp/src/components/SideDrawer/SideDrawer.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.sideDrawer {
```
Comment (nasaku898) [inline bike-erp/src/components/SideDrawer/SideDrawer.css:1]: Same comment as before, change to `Material UI`

**Review by nasaku898 — CHANGES_REQUESTED** [review]
feature looks good and works. Just added a few comments

File: bike-erp/src/components/Menu/ERPMenuStyle.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
+/* .navbar {
```
Comment (nasaku898) [inline bike-erp/src/components/Menu/ERPMenuStyle.ts:1]: remove comment

**Review by nasaku898 — APPROVED** [review]

**Review by Manraj45 — APPROVED** [review]
Good job! Working perfectly!

**Pull request merged**